app-misc/trash-cli/trash-cli-0.23.9.23.ebuild

```diff
@@ -3,25 +3,31 @@
 
 EAPI=8
 
-PYTHON_COMPAT=( python3_{9,10} )
-
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..12} )
 inherit distutils-r1
 
 DESCRIPTION="Python scripts to manipulate trash cans via the command line"
 HOMEPAGE="https://github.com/andreafrancia/trash-cli"
-SRC_URI="https://github.com/andreafrancia/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="
+	https://github.com/andreafrancia/${PN}/archive/${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
```

Comment:
Huh, why did you split this line into multiple ones?

Reply:
Oh, this was probably auto formatter doing something silly.